docs/section15/Rule15-9.md

```diff
@@ -0,0 +1,28 @@
+# Transformers - Rule 15-9
+**Rule ID:** Rule 15-9  
+**Rule Description:** Transformer capacity ratio reported in Baseline RMR equals User RMR  
+**Rule Assertion:** User RMR = Baseline RMR  
+**Appendix G Section:** Transformers  
+**Appendix G Section Reference:**  Table G3.1 15. Distribution Transformers  
+**Data Lookup:** Table 8.4.4  
+**Evaluation Context:**  Each Data Element  
+**Applicability Checks:**
+1. Transformer is DRY_TYPE
+2. U_RMR transformer capacity is within range of Table 8.4.4  
+
```

Comment:
If we change it to B-RMR = P-RMR as suggested above, then we can skip applicability since it would already be checked as part of P-RMR=U-RMR. If we keep it as is (B-RMR=U-RMR, then also need to add a check that U-RMR efficiency exceeds minimums in Table 8.4.4.